### ISSUE TITLE
Do we have examples for grad_clip parameter?

### ISSUE THREAD

**Issue by Cuijie12358** [open-mmlab/mmclassification#216]
Hi, I can see we have donfigs like`optimizer_config = dict(grad_clip=None)`, do we have examples for  `grad_clip`?

**Comment by mzr1996** [comment]
if the `grad_clip` option isn't None, the `OptimizerHook` will call `torch.nn.utils.clip_grad_norm_` function after loss backward.
So, you can refer to [torch docs](https://pytorch.org/docs/stable/generated/torch.nn.utils.clip_grad_norm_.html?highlight=clip%20grad#torch.nn.utils.clip_grad_norm_).
You can use grad_clip like this:
```
optimizer_config = dict(grad_clip=dict(max_norm=0.5))
```
